### PR TITLE
[antd-migration] migrate from plain css to antd

### PR DESCRIPTION
**Pull request by RawandDev**
Will be working on it later. If you want to work on one of them, comment on this pr.
- [x] content.
- [x] KurdCard.
- [x] Dropdown.

### CODE REVIEW

**Comment by mhmd-azeez** [comment]
This PR will probably create a merge conflict with #10. Maybe you should wait until we decide on that because I don't want you to throw away all your code. cc @hezhamen

**Comment by RawandDev** [comment]
Alright, I will wait.

**Comment by hezhamen** [comment]
I agree, I was gonna say we wait for the typescript to be merged.

**Comment by AramRafeq** [comment]
@RawandDev  wait until we merge some breaking changes

**Comment by rojvv** [comment]
@RawandDev now you can sync your fork resolving the conflicts.

**Comment by rojvv** [comment]
@RawandDev you can see the error by running `yarn build`.

**Comment by RawandDev** [comment]
@roj1512 I think this is a typescript error and I don't know much about it. I can't use `align="center"`

image

**Comment by RawandDev** [comment]
Feel free to add any suggestions.

**Comment by rojvv** [comment]
@RawandDev you fixed errors with TS, that's cool.

File: pages/index.tsx
Comment (rojvv) [inline pages/index.tsx]: This text is appearing too small, in my opinion.

File: pages/index.tsx
Comment (rojvv) [inline pages/index.tsx]:
```suggestion
            Meet {awesomeKurds.kurds.length} awesome Kurds.
```

**Review by rojvv — CHANGES_REQUESTED** [review]

**Comment by rojvv** [comment]
@RawandDev you should also resolve the current conflicts.

File: pages/index.tsx
Comment (RawandDev) [reply pages/index.tsx]: I agree, but if you want to make it bigger I guess you need to use vanilla css. I wanted not to use `style={{...}}` and `className={styles.STYLE_NAME}` as much as I could.

**Comment by RawandDev** [comment]
> you fixed errors with TS, that's cool.

Actually, I did a bit of cheating here, I used `style={{...}}` instead of `align="center`. I prefer setting `align` rather than inline styling. It would be nice if you could check it and see if you can fix it with using `align`. It is in the `Footer`.

**Comment by rojvv** [comment]
Sure. Can I get rights to push to your fork?

**Comment by RawandDev** [comment]
> Sure. Can I get rights to push to your fork?

Sure.

**Comment by rojvv** [comment]
Thanks, @hezhamen! <3

**Comment by RawandDev** [comment]
@hezhamen Could you link each pr to its issue? Closing the pr will also close the issue in this way.

**Comment by rojvv** [comment]
@RawandDev the `<Footer>` doesn't have an `align=` argument itself. Hence, your current way is not wrong. But you can still put the footer text inside a `<p>` extender or something, and then align that to center instead of its parent.

**Comment by RawandDev** [comment]
> @RawandDev the `<Footer>` doesn't have an `align=` argument itself. Hence, your current way is not wrong. But you can still put the footer text inside a `<p>` extender or something, and then align that to center instead of its parent.

Yeah, I think it doesn't have `algin` argument but it is working on localhost. I think the current one is good but I have used some inline styles, and not sure whether to go with `style={{..}}` or `className={styles.NAME}`.

**Comment by rojvv** [comment]
@RawandDev as I said, it's better to change it to something like this: `<Footer><Text /* make this align center */>....</Text></Footer>`.

**Comment by rojvv** [comment]
I haven't used antd, so you know better about its components.

**Comment by RawandDev** [comment]
> @RawandDev as I said, it's better to change it to something like this: `<Footer><Text /* make this align center */>....</Text></Footer>`.

I'm also not familiar with antd but as I did a research you can't also use `align` with `Typograph`s.
```
<Footer><Text style={{ textAlign: "center" }}>....</Text></Footer>
```
You will end up doing pretty much the same with more codes.

**Comment by rojvv** [comment]
That's still much better than doing the alignCenter on the Footer.

File: pages/index.tsx
Comment (rojvv) [reply pages/index.tsx]: Maybe use another variant of `Text`? There's also no issue with `style={{...}}` imho.

Comment (RawandDev) [reply pages/index.tsx]: There is not any variant that supports bigger text. I think for this issue if you use `style=...` and make the font bigger, then you need to make it smaller on small screens and you can achieve this in `css` which then you need to have `className={styles....}` and make the code messier. By far this is what I have thought of, feel free to commit to this pr tho.

**Comment by RawandDev** [comment]
Not sure if this pr is ignored. If so, please close it without merging 👀

**Comment by rojvv** [comment]
@RawandDev no, it is not. But a repository can't stop being updated just because of PRs. Conflicts are easy, believe me.

**Comment by AramRafeq** [comment]
Guys there is conflict in one file resolve that and merge it this PR will improve the codebase quality

**Comment by rojvv** [comment]
@AramRafeq are you admin of this repo?

**Comment by rojvv** [comment]
If you're admin, you can resolve the conflicts yourself and then merge it.

**Comment by RawandDev** [comment]
> @RawandDev no, it is not. But a repository can't stop being updated just because of PRs. Conflicts are easy, believe me.

I didn’t say that the repo shouldn’t be updated.
I can resolve conflicts. I know it is not hard but none of the owners seemed to be commenting on this pr recently.

**Comment by RawandDev** [comment]
I will resolve the conflicts when I’m on the pc.

**Comment by AramRafeq** [comment]
> @AramRafeq are you admin of this repo?

Nope I'm a collaborator @hezhamen  please take the lead

**Comment by rojvv** [comment]
@AramRafeq @hezhamen this can get merged now.

**Review by rojvv — APPROVED** [review]

**Comment by RawandDev** [comment]
I removed some stuff. I saw @hezhamen recently added `.dealer` in css but I couldn't find what was the purpose of it. I removed it and some other unused codes.

**Comment by hezhamen** [comment]
.dealer handles our grid for cards

**Comment by rojvv** [comment]
@hezhamen can you fix the broken changes yourself?

**Comment by rojvv** [comment]
Or maybe I try.

**Comment by hezhamen** [comment]
@roj1512 I have made you a collaborator on the repo. You can make the changes yourself. I need to work on some styling issues we currently have.

**Comment by RawandDev** [comment]
> .dealer handles our grid for cards

I haven't seen it being used. I also used antd layout so you probably won't need it. Check the deployed version and see if there is any bug.

**Comment by hezhamen** [comment]
@RawandDev the cards are too big atm, can you make 5 cards appear per row?

**Comment by hezhamen** [comment]
This pr will get merged once someone increases cardPerRow to 5.

**Comment by RawandDev** [comment]
@hezhamen I don't think this is achievable in antd. Antd by default doesn't support 5 column grids as far as I know.

**Comment by rojvv** [comment]
@hezhamen so you should now decide to agree with no 5 per row or not use antd.

**Comment by hezhamen** [comment]
let me check documentation again, and if antd doesn't support it... we'll go with Vanilla CSS for our grid system

**Comment by rojvv** [comment]
They don't. Here's a very good confirmation: https://github.com/ant-design/ant-design/issues/18104.

**Comment by hezhamen** [comment]
you're right, I'll set up vanilla CSS for it

**Comment by RawandDev** [comment]
@roj1512 did you revert the changes then merge the pr? If so, why did you merge this?

**Comment by rojvv** [comment]
@RawandDev I just re-added the original styles, and then modified the cards part of the main page. Tell me if something's wrong.

**Comment by RawandDev** [comment]
@roj1512 I'm confused xD. Which part exactly did you change? It seems like it doesn't even use antd for layout.

There are only two `Layout` components that is used in `index.tsx` and it doesn't affect anything. Other than that, I think everything is just as it was before.

issue with responsive.
https://gyazo.com/c4c4fdf37aae1c8428886eb74ff41a68

**Comment by rojvv** [comment]
@RawandDev can you please send a PR to fix that maintaining the 5 cell per row?

**Pull request merged**
